_overviews/scala3-book/domain-modeling-tools.md

```diff
@@ -408,7 +408,7 @@ Here’s an example of a “string utilities” object that contains a set of me
 ```scala
 object StringUtils {
   def truncate(s: String, length: Int): String = s.take(length)
-  def containsWhitespace(s: String): Boolean = s.object_1es(".*\\s.*")
```

Comment:
There is also [another one](https://github.com/scala/docs.scala-lang/pull/2652/files#diff-b05441bfff28fb2a218502787243f16728be4feecba38fe3465e03fdfe0249c1R423)

Reply:
Done.